packages/beautifulsoup4/meta.yaml

```diff
@@ -0,0 +1,12 @@
+package:
+  name: beautifulsoup4
+  version: 4.7.1
+source:
+  sha256: 945065979fb8529dd2f37dbb58f00b661bdbcbebf954f93b32fdf5263ef35348
+  url: https://files.pythonhosted.org/packages/80/f2/f6aca7f1b209bb9a7ef069d68813b091c8c3620642b568dac4eb0e507748/beautifulsoup4-4.7.1.tar.gz
+requirements:
+  run:
+  - soupsieve
+test:
+  imports:
+  - beautifulsoup4
```

Comment:
I think:

```suggestion
  - BeautifulSoup4
```

Reply:
Updated. Thanks!

🤞

Reply:
Hmmm, I wonder if this should be `bs4` actually?

Reply:
Good point.  I think so.